internal_ws/yktrace/src/lib.rs

```diff
@@ -91,11 +91,3 @@ pub fn start_tracing(kind: TracingKind) -> ThreadTracer {
         }
     }
 }
-
-/// A debugging aid for traces.
-/// Calls to this function are recognised by Yorick and a special debug TIR statement is inserted
-/// into the trace. Interpreter writers should compile-time guard calls to this so as to only emit
-/// the extra bytecodes when explicitely turned on.
-#[inline(never)]
-#[trace_debug]
-pub fn trace_debug(_msg: &'static str) {}
```

Comment:
Does removing this not break the `trace_debug_tir` test?

This should be used.

Reply:
That test uses `ykrt::trace_debug`, not `yktrace::trace_debug`.

https://github.com/softdevteam/yk/blob/19da1faacb1135ea834528ceff75f4ac2c0c49bf/ykrt/src/lib.rs#L9-L15

Reply:
Ah! So this is my fault for leaving it left over. Thanks for spotting it.